person.rb

```diff
@@ -5,7 +5,7 @@ class Person < Nameable
   attr_reader :id
```

Comment:
- [ ] Kindly add the rentals as an `attr_accesor` for the Person class and initialize the rentals array in the initialize method 👍

Reply:
Ive added `:rentals` to attr_reader because there is already a method called `add_rental` to modify the attribute